installer/bundle/bundle_skel_Linux.sh

```diff
@@ -690,8 +569,8 @@ fi
 #
 
 EXIT_STATUS=0
-SCX_EXIT_STATUS=0
 OMI_EXIT_STATUS=0
+SCX_OMI_EXIT_STATUS=0
```

Comment:
Oh, I see why you renamed this. This is now combined due to bundling of operations. No, this makes perfect sense, keep it just the way it is!

Made me wonder what I did in OMS. I just called it KIT_STATUS (since I was installing a bunch of different things). I was fearful I got too lazy and didn't rename it to something meaningful! :disappointed:  But no, we're both good! :smile: